nolearn/lasagne/tests/test_visualize.py

```diff
@@ -55,3 +55,15 @@ def test_plot_occlusion_colored_non_square(
         plot_occlusion(net_color_non_square, X[:3], [3, 2, 1])
         plt.clf()
         plt.cla()
+        
+    def test_draw_to_file(self, net_fitted):
+        import os
+        from nolearn.lasagne.visualize import draw_to_file
+        
+        draw_to_file(net_fitted.get_all_layers(), 'network.pdf', output_shape=False)
+        os.remove('network.pdf')
```

Comment:
It's maybe better to use pytest's [`tmpdir` fixture](https://pytest.org/latest/tmpdir.html) to create the temporary file.  Something like this should work (untested):

```python
    def test_draw_to_file(self, net_fitted, tmpdir):
        from nolearn.lasagne.visualize import draw_to_file
        fn = str(tmpdir.join('network.pdf'))
        draw_to_file(net_fitted.get_all_layers(), fn, output_shape=False)
```